queries.sql

```diff
@@ -32,4 +32,74 @@ SELECT * FROM animals
 /* Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg) */
 SELECT * FROM animals
 	WHERE weight_kg >= 10.4
-		AND weight_kg <= 17.3;
\ No newline at end of file
+		AND weight_kg <= 17.3;
+
+/* 	Update the animals table by setting the species column to digimon for all animals that have a name ending in mon.
+	Update the animals table by setting the species column to pokemon for all animals that don't have species already set.
+	Commit the transaction. */
+BEGIN;		
+UPDATE animals 
+	SET species = CASE
+					WHEN name LIKE '%mon'
+						THEN 'digimon'
+					ELSE
+						'pokemon'
+					END;
+COMMIT;
```

Comment:
* * According to the [requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Inside%20a%20transaction%20update%20the%20animals%20table%20by%20setting%20the%20species%20column%20to%20unspecified.%20Verify%20that%20change%20was%20made.%20Then%20roll%20back%20the%20change%20and%20verify%20that%20the%20species%20columns%20went) you are to 

> Verify that the change was made.

> verify that the change persists after commit 
* HINTS you can do all these by following these steps 
* After setting species , then
* You check to see if species were actually set by running `SELECT species from animals;` 
*  Then commit the transaction and verify that the change persists by running ``SELECT species from animals;``

Reply:
Thanks for the information. I will update it